### ISSUE TITLE
[doc] link to the source

### ISSUE THREAD

**Issue by PhilLAL** [esrf-bliss/Lima#94]
Hi,
in each readthedoc page is a link up-right to the "View page source".

Like in https://tango-controls.readthedocs.io/en/dev/ this link could be updated to a link to Github to help contributors. It could be especially useful if you want contributors to propose themselves some corrections. 

Regards.

**Comment by laurent-claustre** [comment]
Hi Philippe,
again good suggestion. You can improve the doc.
Thanks.

**Comment by sdebionne** [comment]
The thing is Github is now a passive mirror of our ESRF's Gitlab repos so we would rather like contributions on Gitlab (the access is somewhat restricted to some organizations). @PhilLAL Do you have an account on ESRF's Gitlab?

**Comment by PhilLAL** [comment]
I remember I had access to it one year ago, but I can't access it currently.
I asked for sending back my password with my password but I did not receive anything, so I registered but I got the following message: 
```
1 error prohibited this user from being saved:

    Email domain is not authorized for sign-up
```
so I think my account has been erased

**Comment by sdebionne** [comment]
That is correct, sign-up is limited to some domains but many synchrotrons and research facilities are in the  white list. What is the domain of the email address you use to register?
